packages/forms/src/__tests__/addressLookup.spec.tsx

```diff
@@ -40,6 +40,7 @@ const defaultProps: AddressProps = {
 	postcodeLabel: 'Postcode',
 	countryLabel: 'Country',
 	changeAddressButton: 'I need to change the address',
+	onValidatePostCode: null,
```

Comment:
I wonder if its possible to test the new behaviour (assert that the onValidatePostCode method is called if the postcode is changed?).